runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -773,6 +773,28 @@ static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode, TR::SymbolRefer
    return storeValueBaseNode;
    }
 
+static void populatePrexArgInfo(OMR::ValuePropagation *vp, TR_PrexArgInfo *argInfo, TR::Node *node, bool isGlobal, bool trace)
```

Comment:
It's misleading to take `isGlobal` as a parameter here. It's only passed to `getConstraint()`, which treats it as an output, so the argument value is unused